src/processpriv.cpp

```diff
@@ -15,9 +15,10 @@ namespace ex = boost::process::extend;
 
 DLLLOCAL extern const TypedHashDecl* hashdeclMemorySummaryInfo;
 
-#define PROCESS_CHECK(RET) if (!m_process) { xsink->raiseException("PROCESS-CHECK-ERROR", "Process is not initialized"); return (RET); }
-
-#define PROCESS_CHECK_NO_RET if (!m_process) { xsink->raiseException("PROCESS-CHECK-ERROR", "Process is not initialized"); return; }
+DLLLOCAL void processCheckError(ExceptionSink* xsink) {
```

Comment:
you could have define a private method checking for `m_process` returning bool and then if `false` return whatever from the method, but I don't care, this solution is fine as well